src/components/Menu.js

```diff
@@ -154,6 +158,36 @@ function AboutDialog(props) {
                         </a>
                         .
                     </p>
+                    <h3>Get Involved</h3>
+                    <p>If you would like to help in the design or development of this website, read on.</p>
+                    <p>
+                        You can connect with the team on our{" "}
+                        <a
+                            href={"https://macovidvaccines.slack.com/"}
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Slack workspace
+                        </a>.
+                    </p>
```

Comment:
Maybe some disagree, but I'm nervous about making it easy for people to join the Slack workspace. Not that I want it private, but I want to keep it to people who plan on contributing. I'd prefer people request to join via email. Mind updating the copy to reflect that (or making the case for keeping it this way)?

Reply:
I agree with @livgust .  Otherwise, we will be having random users asking "Where can I get a vaccine?" in Slack